planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -150,6 +148,24 @@ class BehaviorPathPlannerNode : public rclcpp::Node
   rclcpp::Publisher<AvoidanceDebugMsgArray>::SharedPtr debug_avoidance_msg_array_publisher_;
   rclcpp::Publisher<MarkerArray>::SharedPtr debug_marker_publisher_;
   void publishDebugMarker(const std::vector<MarkerArray> & debug_markers);
+
+  template <class T>
+  size_t findEgoIndex(const std::vector<T> & points) const
```

Comment:
Memo: The duplication of this function is to reduce input args by using the member variable `planner_data_`.